### ISSUE TITLE
"Soft" mode should support relative line numbers

### ISSUE THREAD

**Issue by srquinn21** [reedes/vim-pencil#57]
In soft mode, `set relative numbers` will give the incorrect movement values. It would be ideal if somehow the number bar could be updated to display the correct movement values.

**Comment by reedes** [comment]
If someone could figure out how to do this, I'd welcome a pull request to fix it.

However I'm not optimistic a fix would be possible, as this seems to be a low-level Vim behavior.

**Comment by alerque** [comment]
I'm pretty sure this would have to be implemented using signs in the gutter rather than the native number column. This would likely not be very per formant. It's going to be super low priority for me to look into but in case anybody finds a way and want to work on this know that I'll be open for reviewing PR's and would be happy to see a contribution that addresses this.

**Comment by antgel** [comment]
I agree that this is difficult / impossible to do (so does [Stack Overflow](https://vi.stackexchange.com/questions/4186/relative-numbers-sidepanel-with-wrapped-lines). Would it be possible to automatically disable relative numbering when soft-wrapping, as it's more hindrance than help?
